### ISSUE TITLE
Best practices for inclusion?

### ISSUE THREAD

**Issue by adamgins** [froala/meteor-froala#4]
Given that contenteditable in Meteor can be tricky  see https://github.com/meteor/meteor/issues/1964 what's the best practice for include froala

I am currently trying it with @Swavek's fix from the link above. It's kinda working...

I am initializing froala with:

     $('.rich-text-editor').editable()

I am currently using a blur to detect changes:

     'blur .rich-text-editor': function (e, editor) { ... 

I then update my field on my document, using ".editable("getHTML")" This seems to work (ie the change shows up in the browser and the db record is changed), the first time around. 

After the update the froala editor seems to be destroyed.

something after this is causing the froala editor to be destroyed. Any ideas, pls?

BTW, I tried initializing and setting .on together (I get similar results):

```
$('.rich-text-editor')
        .editable()
        .on('editable.blur', function (e, editor) {
          
            e.preventDefault();
            e.stopImmediatePropagation();
            console.log("blur")
            var blurredItem = e.currentTarget;
           <SO SOME UPDATE STUFF WITH  $(blurredItem).editable("getHTML")>
        });
```
Basically, it never get's the blur event on the second time.

I should add, I don't see any errors in the console either.

**Comment by adamgins** [comment]
Hi, I ha been using this now... stoked with it. Thanks!!!
